common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -438,6 +445,5 @@ nav.sequence-bottom {
 
 /*  &:focus {
     outline: 0;
-  }
-*/
+*/  }
```

Comment:
All fixed up, thanks @cptvitamin. We might want to revisit (read: cleanup) this file in the future.